<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainScreen</class>
 <widget class="QWidget" name="mainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Infotainment System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/mainBG/media/blurredColors.jpeg);</string>
  </property>
  <widget class="QStackedWidget" name="pagesSwitch">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>990</width>
     <height>482</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="homePage">
    <widget class="QGroupBox" name="iconsGroup">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>41</y>
       <width>800</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background:none;</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="phoneButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>60</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="fmButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="musicButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>60</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="gpsButton">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="bluetoothButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="settingsButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>250</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-radius: 50px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="phonePage">
    <widget class="QLabel" name="phoneLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>571</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButtonPh">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius: 80px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="fmPage">
    <widget class="QPushButton" name="homeButtonFm">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fmLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>571</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>FM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="musicPage">
    <widget class="QPushButton" name="homeButtonMs">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="musicLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>571</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Music</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="gpsPage">
    <widget class="QPushButton" name="homeButtonGp">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="gpsLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>571</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>GPS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="bluetoothPage">
    <widget class="QPushButton" name="homeButtonBl">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="bluetoothLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>571</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Bluetooth</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="settingsPage">
    <widget class="QPushButton" name="homeButtonSt">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>430</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="accessibleName">
      <string notr="true"/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
border-radius: 50px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="settingsLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>571</width>
       <height>291</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="infoGroup">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
background:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="dateText">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>5</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="timeText">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>5</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
